endPoint/nonSecureHttpHostEndPoint.js

```diff
@@ -22,60 +22,70 @@ export default class NonSecureHttpHostEndPoint extends HttpHostEndPoint {
 
   _createServer() {
     return http.createServer(async (req, res) => {
-      try {
-        /** @type {Request} */
-        let cms = null;
-        /**@type {BinaryContent[]} */
-        const fileContents = [];
-        /**@type {NodeJS.Dict<string>} */
-        const formFields = {};
-        /**@type {NodeJS.Dict<string>} */
-        const jsonHeaders = {};
-        const createCmsAndCreateResponseAsync = async () => {
-          cms = await this._createCmsObjectAsync(
-            req.url,
-            req.method,
-            req.headers,
-            formFields,
-            jsonHeaders,
-            req.socket
-          );
-          const result = await this.#service.processAsync(cms, fileContents);
-          const [code, headers, body] = await result.getResultAsync();
-          res.writeHead(code, headers);
-          res.end(body);
-        };
-        if (req.method === "POST") {
-          /**@type {Array<BinaryContent>}*/
-          const bb = busboy({ headers: req.headers });
-          bb.on("file", (name, file, info) => {
-            const ContentParts = [];
-            file.on("data", (x) => ContentParts.push(x));
-            file.on("end", async () => {
-              const content = new BinaryContent();
-              content.url = `${req.headers["host"]}${req.url}`;
-              content.mime = info.mimeType.toLowerCase();
-              content.name = info.filename;
-              content.payload = Buffer.concat(ContentParts);
-              fileContents.push(content);
+      bodyParser.json()(req, res, (err) => {
```

Comment:
In other case (non json content) this code work? it is tested?

Reply:
yes; it works on non secure version of fingerfood but needs to write some unit tests for it to make sure